WalletWasabi/Services/WasabiSynchronizer.cs

```diff
@@ -44,19 +38,22 @@ public class WasabiSynchronizer : NotifyPropertyChangedBase, IFeeProvider
 
 		public WasabiSynchronizer(Network network, BitcoinStore bitcoinStore, WasabiClient client)
 		{
-			CreateNew(network, bitcoinStore, client);
+			Network = Guard.NotNull(nameof(network), network);
+			WasabiClient = Guard.NotNull(nameof(client), client);
+			LastResponse = null;
+			BitcoinStore = Guard.NotNull(nameof(bitcoinStore), bitcoinStore);
+			_running = 0;
+			Cancel = new CancellationTokenSource();
 		}
 
-		public WasabiSynchronizer(Network network, BitcoinStore bitcoinStore, Func<Uri> baseUriAction, EndPoint torSocks5EndPoint)
+		public WasabiSynchronizer(Network network, BitcoinStore bitcoinStore, Func<Uri> baseUriAction, EndPoint torSocks5EndPoint) 
```

Comment:
This constructor makes no sense. Please remove it.

Reply:
Removed.